documentation/api/definitions.md

```diff
@@ -87,6 +87,48 @@ Enumeration that describes the current state of the collection rule.
 | `Throttled` | Indicates that the collection rule is temporarily throttled because the ActionCountLimit has been reached within the ActionCountSlidingWindowDuration. |
 | `Finished` | Indicates that the collection rule has completed and will no longer trigger. |
 
+## CapturedMethod
+
+First Available: 9.0 Preview 3
+
+Object describing a captured method and its parameters.
+
+| Name | Type | Description |
+|---|---|---|
+| `requestId` | guid | Unique identifier for the capture request operation. |
```

Comment:
We shouldn't expose this, it's an implementation detail and doesn't mean anything to the consumer of this API. Captured parameters will inherently be grouped by "capture request operation" by the fact that they are being egressed together to the same destination.